extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -333,6 +337,39 @@ export class Bind {
     return this.history_;
   }
 
+  /**
```

Comment:
"Calls setState(s), where s is data.state with non-overridable keys removed."

Reply:
Done